modules/context/context.go

```diff
@@ -290,7 +290,7 @@ func (ctx *Context) Header() http.Header {
 // FIXME: We should differ Query and Form, currently we just use form as query
 // Currently to be compatible with macaron, we keep it.
 
-// Query returns request form as string with default
+// Query returns request form (NOT URL.Query) as string with default
```

Comment:
what does this meant to bee?

Reply:
The name `Query` suggests that you would get back a standard `URL.Query()`, but it is actually `Form`. The existing language wasn't clear to me and I thought it was BOTH Query and Form (like PHP does), which is not the case.

Reply:
I have send another PR to resolve the problem totally. see #16562